RastenbergerAngularJS/src/main/webapp/foodoffers.js

```diff
@@ -0,0 +1,54 @@
+var app = angular.module("foodoffer-directives", []);
```

Comment:
In this module two "directives" are declared, so they can be used as html elements in our index.html.
```html
<foodoffer-list></foodoffer-list>
<foodoffer-form></foodoffer-form>
```